aspnet/publishing/iis.rst

```diff
@@ -291,6 +312,17 @@ Troubleshooting
 
 - Confirm that the Application Pool is not in the `Stopped` state.
 
+IISIntegration module was not injected
```

Comment:
Will do: I see now that these are called out as "middleware" here: https://docs.asp.net/en/latest/fundamentals/middleware.html